.changeset/tidy-baboons-rest.md

```diff
@@ -0,0 +1,5 @@
+---
+"wrangler": minor
```

Comment:
Wouldn't this be a `major` change, since it's breaking an existing API? Users who upgrade to the version this is published in who use the current environment variable would have their local/CI setup break.

Reply:
We are not dropping support of the old name yet. There will be a warning message to suggest users using the CLOUDFLARE version but the old name will continue to be accepted.

Reply:
Oh I see that now, thanks!